src/Equations/Fluid/TritiumRateTerm.cpp

```diff
@@ -0,0 +1,82 @@
+#include "DREAM/Equations/Fluid/TritiumRateTerm.hpp"
+
+/**
+ * Implementation of a class which represents the Tritium runaway rate contribution to the n_re equation.
+ * Employs the analytical generation rate calculated by RunawayFluid.
+ */
+using namespace DREAM;
+
+/**
+ * Constructor.
+ *
+ * g:           Grid on which this term lives.
+ * uqn:         UnknownQuantityHandler object.
+ * iIon:        Index of ion to which this term should be applied.
+ * rf:          RunawayFluid object.
+ * ions:        IonHandler object.
+ * scaleFactor: Factor by which to rescale the equation term.
+ */
+TritiumRateTerm::TritiumRateTerm(
+    FVM::Grid *g, FVM::UnknownQuantityHandler *uqn, const len_t iIon,
+    RunawayFluid *rf, IonHandler *ions, real_t scaleFactor
+) : IonEquationTerm<FVM::EquationTerm>(g, ions, iIon), RunawaySourceTerm(g,uqn),
+    REFluid(rf), scaleFactor(scaleFactor) { }
+
+
+/**
+ * Set elements of the Jacobian corresponding to this equation term.
+ */
+void TritiumRateTerm::SetCSJacobianBlock(
+    const len_t uqtyId, const len_t derivId, FVM::Matrix *jac,
+    const real_t*,
+    const len_t iIon, const len_t Z0, const len_t rOffset
+) {
+    if (uqtyId == derivId)
+        this->SetCSMatrixElements(jac, nullptr, iIon, Z0, rOffset);
+}
```

Comment:
plans for off-diagonal contributions analogous to the compton term? think it will be slightly simpler for this term